CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * Update API endpoints on UI due to changes on backend across the application (UIDATIMP-112, UIDATIMP-113, UIDATIMP-114)
 * Implement "Save the new file extension" feature (UIDATIMP-79)
 * Implement return to Choose Job Profile page functionality (UIDATIMP-103)
+* Write tests for return to Choose Job Profile page functionality (UIDATIMP-154)
```

Comment:
Let's have the name of the commit like this instead of just `UIDATIMP-154: Write tests`:

`UIDATIMP-154: Write tests for return to Choose Job Profile page functionality`